### DISCUSSION TITLE
We need your testimonial words

### DISCUSSION THREAD

**Discussion by sonvnn** [templaza/astroid-framework#90] [General]
Hi @Chacapamac , @Joomlaplates & everyone

Please give me your testimonial about Astroid Framework. I will add them to https://astroidframe.work/

Thank you very much.

Kind Regards,
Sonny

**Comment by Chacapamac** [comment]
I worked with Joomla from the 1.5 eras. Last two years, I thoroughly tested  ALL the top Joomla Frameworks to replace my trusted RocketTheme that didn’t respond anymore to what I demanded of a Joomla Framework. As a result, I find Astroid to be the most modern & flexible Joomla-friendly framework to date, from far.

**Comment by sonvnn** [comment]
Hi @Joomlaplates

I am back here. I still need your testimonial and Joomlaplates info to add to Astroidframe.work site.

Kind Regards,
Sonny

**Reply by Chacapamac** [reply]
**Welcome back Sonny!** 
Hope you are feeling better. 
I will look at the site again tomorrow and come back with ideas. As I was saying I’m personally extremely busy for the next month or so ,but I will still find time to help. I should give way more time to the project after that.

**Comment by joomlaplates-de** [comment]
Hi Sonny, welocme back!!!

here is my review with avatar, but I already send it to you via Mail

_After Warp 7 is not supported anymore, we were looking for another Joomla Theme Framework. And finally after a long time of research we tried Astroid. We are impressed about the flexibility and user friendly options. Our clients are happy and currently we have already build more than 50 Joomla 4 Templates with it._
https://drive.google.com/file/d/1ejTy4Tbprly1napluSMKP9OFJTF6v8u1/view?usp=sharing

**Comment by IV1UK** [comment]
I was an Astroid user through JD templates for many years, Astroid had incredible potential as a real leader for the Joomla community, especially for websites built for client end-user services, a clean and professional teamplate designer, I am so happy to see after JoomDev dropped the community on it's backside that some amazing Joomla community developers have kept Astroid alive. Joomla is without doubt the best CMS bar none, Astroid is another reason to continue to support it and an awesome community that have loved Joomla since the beginning. Long live Joomla and Astroid should be built into it install package, keep up the awesome work guys!
